common/app/views/fragments/atoms/quiz.scala.html

```diff
@@ -8,54 +8,34 @@
     <h2>@quiz.title</h2>
 }
 <form action="@postUrl(quiz)" method="POST">
-
-    <ol class="quiz @{if (quiz.quizType == "personality") "quiz--personality" else "quiz--knowledge" } @{if (showResults) "quiz--results" else "quiz--questions"}">
+    <ol class="quiz quiz--@{quiz.quizType} @{if (showResults) "quiz--results" else "quiz--questions"}">
         @quiz.content.questions.zipWithIndex.map { case (question, index) =>
-            @renderQuestion(question, playForm(s"answers[$index]"), maybeResults.flatMap(_.getAnswerFor(question)))
+            @renderQuestion(question,
+                playForm(s"answers[$index]"),
+                maybeResults.flatMap(_.getAnswerFor(question))
+            )
         }
     </ol>
-
     @maybeResults.map { results =>
         @renderEndMessage(results)
     }.getOrElse {
-        <button class="button" type="submit">Submit answers</button>
+        <button class="button button--xlarge button--primary" type="submit">Submit answers</button>
     }
-
 </form>
 
-@renderEndMessage(results: QuizResults) = {
-    <div class="quiz__end-message">
-        <div class="quiz__score-message">
-            @if(results.isKnowledge) {
-                You got <span class="quiz__score">@results.knowledgeScore/@quiz.content.questions.size</span>
-            }
-            @if(results.isPersonality) {
-                @results.resultBucket.map(_.description)
-            }
-        </div>
-        @results.resultGroup.map { resultGroup =>
-            <div class="quiz__bucket-message">@resultGroup.title</div>
-        }
-        <div class="quiz__share">
-            <div class="quiz__share__cta">Share your results</div>
-            @fragments.social(sharelinks)
-        </div>
-    </div>
-}
-
-@renderQuestion(question: Question, field: play.api.data.Field, maybeInputAnswer: Option[Answer]) = {
+@renderQuestion(question: Question, field: play.api.data.Field, maybeUserAnswer: Option[Answer]) = {
     <li class="quiz__question
         @{ if (showResults) {
-            s"quiz__question--${if(maybeInputAnswer.isEmpty) "without-response" else "with-response" }"
+            s"quiz__question--${if(maybeUserAnswer.isEmpty) "without-response" else "with-response" }"
           }
         }">
-        <span class="quiz__question__label">@question.text</span>
+        <span class="quiz__question-text">@question.text</span>
 
         <div class="quiz__question__rows">
```

Comment:
Can we do `.quiz__rows` and then `.quiz__rows .quiz__rows--question`. Should we even call the modifier `--question`. Can we use something more vague like `.quiz__rows--small`. Then we can use it anywhere without coupling it with `question`.